ExceptionCollection error

```
[2019-04-25 10:47:13.180] INFO Engine - Finished CPE Analyzer (0 seconds)
[2019-04-25 10:47:13.181] INFO Engine - Finished False Positive Analyzer (0 seconds)
[2019-04-25 10:47:13.184] INFO Engine - Finished NVD CVE Analyzer (0 seconds)
[2019-04-25 10:47:13.207] INFO Engine - Finished Vulnerability Suppression Analyzer (0 seconds)
[2019-04-25 10:47:13.213] INFO Engine - Finished Dependency Bundling Analyzer (0 seconds)
[2019-04-25 10:47:13.247] INFO Engine - Analysis Complete (1 seconds)
Exception in thread "main" Syntax error compiling at (/tmp/form-init12567659599026164559.clj:1:74).
	at clojure.lang.Compiler.load(Compiler.java:7647)
	at clojure.lang.Compiler.loadFile(Compiler.java:7573)
	at clojure.main$load_script.invokeStatic(main.clj:452)
	at clojure.main$init_opt.invokeStatic(main.clj:454)
	at clojure.main$init_opt.invoke(main.clj:454)
	at clojure.main$initialize.invokeStatic(main.clj:485)
	at clojure.main$null_opt.invokeStatic(main.clj:519)
	at clojure.main$null_opt.invoke(main.clj:516)
	at clojure.main$main.invokeStatic(main.clj:598)
	at clojure.main$main.doInvoke(main.clj:561)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.main.main(main.java:37)
Caused by: org.owasp.dependencycheck.exception.ExceptionCollection: One or more exceptions occurred during dependency-check analysis
	The download was interrupted; unable to complete the update
	at org.owasp.dependencycheck.Engine.analyzeDependencies(Engine.java:693)
	at nvd.task.check$scan_and_analyze.invokeStatic(check.clj:43)
	at nvd.task.check$scan_and_analyze.invoke(check.clj:38)
	at nvd.task.check$_main.invokeStatic(check.clj:54)
	at nvd.task.check$_main.invoke(check.clj:51)
	at user$eval678.invokeStatic(form-init12567659599026164559.clj:1)
	at user$eval678.invoke(form-init12567659599026164559.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7176)
	at clojure.lang.Compiler.eval(Compiler.java:7166)
	at clojure.lang.Compiler.load(Compiler.java:7635)
	... 12 more
Subprocess failed
```

running 1.0.0

--- a/.gitignore
+++ b/.gitignore
@@ -12,6 +12,9 @@ pom.xml.asc
 .nrepl-port
 test/resources/dc.h2.db
 test/resources/dc.trace.db
+test/resources/odc.mv.db
 test/resources/jsrepository.json
 .idea
 *.iml
+cache/
+oss_cache/

--- a/project.clj
+++ b/project.clj
@@ -24,7 +24,7 @@
       :global-vars {*warn-on-reflection* true}
       :plugins [
         [lein-cljfmt "0.6.4"]
-        [lein-codox "0.10.6"]
+        [lein-codox "0.10.7"]
         [lein-cloverage "1.1.1"]]
       :dependencies [
         [commons-collections "3.2.1"]]}})

--- a/project.clj
+++ b/project.clj
@@ -5,11 +5,11 @@
     :name "The MIT License (MIT)"
     :url "http://opensource.org/licenses/MIT"}
   :dependencies [
-    [org.clojure/clojure "1.10.0"]
+    [org.clojure/clojure "1.10.1"]
     [clansi "1.0.0"]
     [org.clojure/data.json "0.2.6"]
     [org.slf4j/slf4j-simple "1.7.26"]
-    [org.owasp/dependency-check-core "4.0.2"]
+    [org.owasp/dependency-check-core "5.0.0"]
     [rm-hull/table "0.7.0"]]
   :scm {:url "git@github.com:rm-hull/lein-nvd.git"}
   :source-paths ["src"]

--- a/src/nvd/config.clj
+++ b/src/nvd/config.clj
@@ -32,7 +32,6 @@
 
 (def ^:private string-mappings
   {Settings$KEYS/ANALYZER_NEXUS_URL [:analyzer :nexus-url]
-   Settings$KEYS/ANALYZER_ASSEMBLY_MONO_PATH [:analyzer :path-to-mono]
    Settings$KEYS/SUPPRESSION_FILE [:suppression-file]
    Settings$KEYS/ADDITIONAL_ZIP_EXTENSIONS [:zip-extensions]
    Settings$KEYS/PROXY_SERVER [:proxy :server]
@@ -45,11 +44,7 @@
    Settings$KEYS/DB_DRIVER_PATH [:database :driver-path]
    Settings$KEYS/DB_CONNECTION_STRING [:database :connection-string]
    Settings$KEYS/DB_USER [:database :user]
-   Settings$KEYS/DB_PASSWORD [:database :password]
-   Settings$KEYS/CVE_MODIFIED_12_URL [:cve :url-1.2-modified]
-   Settings$KEYS/CVE_MODIFIED_20_URL [:cve :url-2.0-modified]
-   Settings$KEYS/CVE_SCHEMA_1_2 [:cve :url-1.2-base]
-   Settings$KEYS/CVE_SCHEMA_2_0 [:cve :url-2.0-base]})
+   Settings$KEYS/DB_PASSWORD [:database :password]})
 
 (def ^:private integer-mappings
   {Settings$KEYS/CVE_CHECK_VALID_FOR_HOURS [:cve :valid-for-hours]})

--- a/src/nvd/report.clj
+++ b/src/nvd/report.clj
@@ -48,6 +48,9 @@
     (.write rg ^String output-dir ^String output-fmt)
     project))
 
+(defn- score [^Vulnerability vulnerability]
+  (.getScore (.getCvssV2 vulnerability)))
+
 (defn- severity [cvssScore]
   (cond
     (= cvssScore 0) :none
@@ -66,7 +69,7 @@
     (style "OK" :green :bright)
     (s/join ", "
             (for [^Vulnerability v (.getVulnerabilities dep)
-                  :let [color (-> (.getCvssScore v) severity color)]]
+                  :let [color (-> v score severity color)]]
               (style (.getName v) color :bright)))))
 
 (defn- vulnerabilities [project ^Engine engine]
@@ -78,7 +81,7 @@
 (defn- scores [^Engine engine]
   (flatten (for [^Dependency dep (.getDependencies engine)
                  ^Vulnerability vuln (.getVulnerabilities dep)]
-             (.getCvssScore vuln))))
+             (score vuln))))
 
 (defn print-summary [project]
   (let [^Engine engine (:engine project)

--- a/test/nvd/config_test.clj
+++ b/test/nvd/config_test.clj
@@ -26,7 +26,7 @@
    [clojure.test :refer :all]
    [nvd.config :refer :all]))
 
-(def dependency-check-version "4.0.2")
+(def dependency-check-version "5.0.0")
 
 (deftest check-app-name
   (is (= "unknown" (app-name {:nome "hello-world" :version "0.0.1"})))

